ts/parts-map/HeatmapSeries.ts

```diff
@@ -293,6 +320,28 @@ seriesType<Highcharts.HeatmapSeries>(
             options.pointRange = pick(options.pointRange, options.colsize || 1);
             // general point range
             this.yAxis.axisPointRange = options.rowsize || 1;
+
+            // Bind new symbol names
+            extend(symbols, {
+                ellipse: symbols.circle,
+                rect: symbols.square
+            });
+        },
+        getSymbol: seriesTypes.scatter.prototype.getSymbol,
+
+        /**
+         * @private
+         * @function Highcharts.seriesTypes.heatmap#setClip
+         * @return {void}
+         */
+        setClip: function (
+            this: Highcharts.HeatmapSeries
+        ): void {
+            Series.prototype.setClip.apply(this, arguments);
+            if (this.options.clip !== false && this.markerGroup) {
```

Comment:
Hm.. isn't `markerGroup` always defined once you made the transition to markers?

Reply:
It was done because of TypeScript definitions. The `Series.markerGroup` is optional, so can be `undefined`. I assumed that would be better to check `this.markerGroup` exists instead of casting it `as any`.
If it is not correct, I can also override the `markerGroup` property within `HeatmapSeries` interface, and make it required. What do you think?

Reply:
It's ok 👍